File: MEMO.MobilClient/app/src/main/java/hu/mobilclient/memo/model/authentication/Registration.kt
package hu.mobilclient.memo.model.authentication

import com.google.gson.annotations.SerializedName
import hu.mobilclient.memo.helpers.Constants

class Registration(@SerializedName("userName")var UserName: String = Constants.EMPTY_STRING,
                   @SerializedName("email")var Email: String = Constants.EMPTY_STRING,
                   @SerializedName("password")var Password: String = Constants.EMPTY_STRING)

File: MEMO.BLL/Authentication/Registration.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace MEMO.BLL.Authentication
{
    public class Registration
    {
        [MaxLength(15)]
        public string UserName { get; set; }

        [MaxLength(50)]
        public string Email { get; set; }

        [MaxLength(20)]
        public string Password { get; set; }
    }
}
